src/Raven.Server/Config/Attributes/ConfigurationEntryAttribute.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Raven.Server.Config.Attributes
 {
-    [AttributeUsage(AttributeTargets.Property)]
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = true)]
```

Comment:
If you want to go that way, then please validate (ConfigurationCategory.Initialize) that all of the attributes have the same scope.

Reply:
Fixed